v03_pipeline/lib/reference_datasets/reference_dataset.py

```diff
@@ -0,0 +1,93 @@
+from enum import Enum
+from types import MappingProxyType
+
+from v03_pipeline.lib.model import AccessControl, DatasetType, Env, ReferenceGenome
+from v03_pipeline.lib.reference_datasets.cadd import load_cadd_ht_from_raw_dataset
+from v03_pipeline.lib.reference_datasets.hgmd import (
+    HGMD_ENUM_SELECT,
+    download_and_import_hgmd_vcf,
+)
+
+LOAD_DATASET_FUNC = 'load_parsed_dataset_func'
+VERSION = 'version'
+RAW_DATASET_PATH = 'raw_dataset_path'
+ENUM_SELECT = 'enum_select'
+
+
+class ReferenceDataset(str, Enum):
+    cadd = 'cadd'
+    hgmd = 'hgmd'
+
+    CONFIG = MappingProxyType(
+        {
+            'cadd': {
```

Comment:
I think you can actually directly use `cadd` here as the variable not as the string

Reply:
I think we can't reference the enum value until it's created.  It's probably better to lift `CONFIG` off the enum for memory reasons any ways, placing it below `ReferenceDataset` in the module so we can use the enum value.

Reply:
CONFIG is now no longer an attribute of the enum and uses the enums as keys. 

@bpblanken did you leave a comment somewhere about namedtuples?